include/rocksdb/cache.h

```diff
@@ -255,12 +255,19 @@ class Cache {
   // returns the memory size for the entries residing in the cache.
   virtual size_t GetUsage() const = 0;
 
+  // returns the memory size for the high priority entries residing in the
+  // cache.
+  virtual size_t GetHighPriorityPoolUsage() const = 0;
+
   // returns the memory size for a specific entry in the cache.
   virtual size_t GetUsage(Handle* handle) const = 0;
 
   // returns the memory size for the entries in use by the system
   virtual size_t GetPinnedUsage() const = 0;
 
+  // returns the number of entries in the cache
+  virtual size_t GetEntries() const = 0;
```

Comment:
I'm not sure this information is necessarily available in the cache. I'm not even sure the value returned by LRU cache and clock cache is accurate, and the information might be unavailable from another implementation. Do you really need this stat?